### ISSUE TITLE
Revert SDL2/OpenGL support for N64, decouple it from app

### ISSUE THREAD

**Issue by giongto35** [giongto35/cloud-game#222]
The introduction of N64 is here: https://github.com/giongto35/cloud-game/pull/195/files . It's awesome to see that Nintendo 64 can run on the system.
https://github.com/giongto35/cloud-game/pull/205/files

However, it also brings high maintenance cost + installation complexity with OpenGL, LibSDL to serve a small usecase. Therefore, what I will do is
1) Revert the LibSDL + OpenGL
2) Decouple Game + Encoding system
3) Run N64 emulator separately

**Comment by sergystepanov** [comment]
So you've decided to remove Libretro from the workers just leaving three (AV+data) "channels" open for some IPC method of communication with an external emulator or app?

**Comment by giongto35** [comment]
Hi @sergystepanov, long time no see.
Yes, the Libretro emulator can be run as a dependent service and communicate over IPC. It doesn't need to be removed totally, just replace some gochannel communication to IPC.
Currently, I made some good progress to interact with exe file using wine in headless mode as well. I will tell you when it is done.
P/S: Regarding the IPC, I intended to use Unix Socket but it is not for Window build. Maybe I should just use normal tcp socket for IPC.
